Update networks doc to avoid passive voice & reduce word count (#6619)
ethereum/ethereum-org-website: src/content/developers/docs/networks/index.md

sidebar: true
---

Networks are different Ethereum environments you can access for development, testing, or production use cases. Since Ethereum is a protocol, there can be multiple independent "networks" that conform to the protocol without interacting with each other.

Your Ethereum account will work across the different networks, but your account balance and transaction history won't carry over from the main Ethereum network. For testing purposes, it's useful to know which networks are available and how to get testnet ETH to play around with.

## Prerequisites {#prerequisites}

You should understand the [basics of Ethereum](/developers/docs/intro-to-ethereum/) before reading up on the different networks, as the test networks will give you a cheap, safe version of Ethereum to play around with.

## Public networks {#public-networks}

> Public networks are accessible to anyone in the world with an internet connection. Anyone can read or create transactions on a public blockchain and validate the transactions being executed. A consensus of peers decides on agreement of transactions and the state of the network.

Reviewer: ```suggestion
Public networks are accessible to anyone in the world with an internet connection. Anyone can read or create transactions on a public blockchain and validate the transactions being executed. The consensus among peers decides on the inclusion of transactions and the state of the network.
```

In addition to Mainnet, there are public testnets. These are networks used by protocol developers or smart contract developers to test both protocol upgrades as well as potential smart contracts in a production-like environment before deployment to Mainnet. Think of this as an analog to production versus staging servers.

> We advise you to test any contract code you write on a testnet before deploying to the Mainnet. Among dapps that integrate with existing smart contracts, most projects have copies deployed to testnets.

Reviewer: ```suggestion
You should test any contract code you write on a testnet before deploying to Mainnet. Among dapps that integrate with existing smart contracts, most projects have copies deployed to testnets.
```

Author: yours  is better 👍

Most testnets use a proof-of-authority consensus mechanism. This means a small number of nodes are chosen to validate transactions and create new blocks – staking their identity in the process. Testnets do not incentivize proof-of-work mining, which can leave them vulnerable.

ETH on testnets has no real value; therefore, there are no markets for testnet ETH. Since you need ETH to actually interact with Ethereum, most people get testnet ETH from faucets. Most faucets are webapps where you can input an address which you request ETH to be sent to.

#### Sepolia {#sepolia}

> A proof-of-work testnet that has the best available like-for-like representation of Ethereum.

Reviewer: ```suggestion
A proof-of-work testnet—the best like-for-like representation of Ethereum Mainnet.
```

### Consortium networks {#consortium-networks}

> The consensus process is controlled by a pre-defined set of nodes that are trusted -- for example, a private network of known academic institutions that each govern a single node, and blocks are validated by a threshold of signatories within the network.

Reviewer: ```suggestion
The consensus process is controlled by a pre-defined set of nodes that are trusted. For example, a private network of known academic institutions that each govern a single node, and blocks are validated by a threshold of signatories within the network.
```
